functions/scheduleinstance/index.js

```diff
@@ -81,25 +94,34 @@ exports.stopInstancePubSub = (event, callback) => {
     const payload = _validatePayload(
       JSON.parse(Buffer.from(pubsubMessage.data, 'base64').toString())
     );
-    compute
-      .zone(payload.zone)
-      .vm(payload.instance)
-      .stop()
-      .then(data => {
-        // Operation pending.
-        const operation = data[0];
-        return operation.promise();
-      })
-      .then(() => {
-        // Operation complete. Instance successfully stopped.
-        const message = 'Successfully stopped instance ' + payload.instance;
-        console.log(message);
-        callback(null, message);
-      })
-      .catch(err => {
-        console.log(err);
-        callback(err);
+    const options = {
+      filter: 'labels.' + payload.label,
```

Comment:
Use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)? e.g.
```
filter: `labels.${payload.label}`
```

Also, can this variable declaration be compressed into one line?

Reply:
Done.